src/Data/StringHasher.cs

```diff
@@ -73,7 +74,7 @@ public string Hash(string input, bool hyphens = true, bool lowercase = false)
 
         private HashAlgorithm GetAlgorithm()
         {
-            return HashAlgorithm.Create(_algorithm);
+            return _algorithm ??= HashAlgorithm.Create(_algorithmName);
```

Comment:
Yep, good catch! 

The issue here is [`HashAlgorithm`](https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.hashalgorithm?view=net-6.0) (the type of the `_algorithm` variable) is a _disposable_ type, you can read more about that here: https://learn.microsoft.com/en-us/dotnet/standard/garbage-collection/using-objects

That's why we have the `using var algo = GetAlgorithm();` line, the `using` statement wraps a disposable object and releases any resources when it is done. 

However since we were caching the object, it was already marked as disposed from a previous call/test, hence this failure. Fixed in [d29a371](https://github.com/cal-itp/hashfields/pull/17/commits/d29a371d97d5cdf861df2ebdedfea30d14cc16e1) by just removing the caching.